lib/metanorma/cli/commands/config.rb

```diff
@@ -0,0 +1,122 @@
+require "pathname"
+
+require "metanorma/cli/stringify_all_keys"
+
+module Metanorma
+  module Cli
+    module Commands
+      class Config < Thor
+        class_option :global, aliases: "-g", type: :boolean, default: false, desc: "Use global config"
+
+        desc "get NAME", "Get config value"
+        def get(name = nil)
+          config_path = Metanorma::Cli.config_path(options.global)
+          config = load_config(config_path)
+
+          if name.nil?
+            print File.read(config_path, encoding: "utf-8")
+          else
+            print(config.dig(*dig_path(name)) || "nil")
```

Comment:
What do you think about using [CLI::UI.say](https://github.com/metanorma/metanorma-cli/blob/master/lib/metanorma/cli/ui.rb#L10) instead of `print`?

Reply:
Agree

Reply:
Fixed